src/elements/content-sidebar/activity-feed/comment/BaseComment.js

```diff
@@ -210,7 +210,9 @@ export const BaseComment = ({
                         user={createdByUser}
                         badgeIcon={
                             annotationActivityLink && (
-                                <IconAnnotation title={<FormattedMessage {...messages.annotationBadge} />} />
```

Comment:
Maybe it's worth checking with products if we want to update this title? They probably didn't know it existed as the icon wasn't focusable. If we need to update then `annotationBadge` should also be removed from messages if it's not used anywhere else

Reply:
this is unused part of code since this icon was not focusable and now we use another one